en/config/actuators.md

```diff
@@ -1,65 +1,170 @@
-This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+Open the view in *QGroundControl* here: **"Q" (app menu) > Vehicle Setup > Actuators** (tab).
+
+## Overview
+
+The view has three sections:
+- [Geometry](#geometry): Set any configurable geometry-related parameters for the [selected airframe](../config/airframe.md).
+  This includes the number and relative position of motors, and the number, function, and properties of [control surfaces](#control-surfaces-geometry) (ailerons, rudders, etc.).
+- [Actuator Outputs](): Assign motors, control surfaces, and other actuators to specific flight controller outputs.
+- [Actuator Testing](): Test that motors and actuators are mapped to the correct outputs, and move as expected.
 
-The multicopter configuration screen looks like this.
+A quadcopter might have an setup screen similar to the one shown below.
+This defines a 4 motor copter with X-geometry, and maps each of the motors to DShot ESC.
+It also maps PWM400 AUX outputs for controlling a parachute and landing gear.
 
 ![Actuators MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_aux.png)
 
-Note that some settings are hidden unless you select the **Advanced** checkbox in the top right corner.
+Note that only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner to display all settings.
+
 
 ## Geometry
 
-The geometry section is used to configure any additional geometry-related settings for the selected [airframe](../config/airframe.md).
-The UI displays a customised view for the selected type; if there are no configurable options this may display a static image of the frame geometry, or nothing at all.
+The geometry section is used to set any configurable geometry-related parameters for the selected [airframe](../config/airframe.md).
+This includes the number and position of [motors](#motor-geometry), and the number, function, and properties of [control surfaces](#control-surfaces-geometry).
+For VTOL tiltrotor vehicles, it will also include the number and properties of [tilt servos](#motor-tilt-servo-geometry)
+
+The UI is customised for the selected airframe.
+Only motors and/or control surfaces that are relevant to the airframe type are displayed or available to assign.
+If there are no configurable geometry options for a particular frame, this section could conceivably be empty.
+
+:::note
+Not all aspects of a geometry are necessarily configurable through this interface.
+Factors that cannot be configured are hard coded for the current airframe. 
+:::
+
+### Motor Geometry
+
+The motor geometry section lets you set the number motors, and the relative position, function, and other properties of each motor.
+The motor properties are similar for all frames, but there are some exceptions.
```

Comment:
What kind of 'exceptions' are there?

Reply:
The ones that have different behaviours, outlined in the following sections. So for example, tiltotors additionally set the associated tilt servo, and standard quads have a direction applied to the motors.

Reply:
Aha gotcha 🙆‍♂️